spring-cloud-stream/src/main/java/org/springframework/cloud/stream/converter/CompositeMessageConverterFactory.java

```diff
@@ -51,7 +51,7 @@ public CompositeMessageConverterFactory() {
 	/**
 	 * @param customConverters a list of {@link AbstractFromMessageConverter}
 	 */
-	public CompositeMessageConverterFactory(List<? extends AbstractFromMessageConverter> customConverters,
+	public CompositeMessageConverterFactory(List<? extends MessageConverter> customConverters,
```

Comment:
With the introduction of `TargetMimeTypeMessageConverter`, are we enforcing any custom message converters to implement this interface instead of extending `AbstractFromMessageConverter`? Also, I think the `ChannelBindingServiceConfiguration` requires this change to have 
```
@Autowired(required = false)
	private List<TargetMimeTypeMessageConverter> customMessageConverters;

```
If I understand it right, we aren't injecting any `MessageConverter` beans in the context instead only the `TargetMimeTypeMessageConverter` beans?